packages/react-proptypes/src/index.ts

```diff
@@ -411,6 +411,10 @@ export const itemShorthandWithKindProp = (kindPropValues: string[]) => {
     ]),
   ])
 }
+export const itemShorthandWithoutJSX = every([
+  disallow(['children']),
+  PropTypes.oneOfType([PropTypes.func, PropTypes.number, PropTypes.object, PropTypes.string]),
```

Comment:
Can we extract this and reuse it in both `itemShorthand` and `itemShorthandWithoutJSX`?

Reply:
These types are different, in `itemShorthand` it's `PropTypes.node`

Reply:
Got it, then should we add these values as well: `boolean | null | undefined` too?

Reply:
Good catch, we definitely should handle `false` as a valid value:
```diff
+PropTypes.oneOf([false])
```